packages/plugins/response-cache/src/plugin.ts

```diff
@@ -133,17 +169,21 @@ export function useResponseCache({
   enabled,
   ignoredTypes = [],
   ttlPerType = {},
-  ttlPerSchemaCoordinate,
+  ttlPerSchemaCoordinate = {},
   idFields = ['id'],
   invalidateViaMutation = true,
   buildResponseCacheKey = defaultBuildResponseCacheKey,
   getDocumentStringFromContext = defaultGetDocumentStringFromContext,
+  shouldCacheResult = defaultShouldCacheResult,
   // eslint-disable-next-line dot-notation
   includeExtensionMetadata = typeof process !== 'undefined' ? process.env['NODE_ENV'] === 'development' : false,
 }: UseResponseCacheParameter = {}): Plugin {
   const ignoredTypesMap = new Set<string>(ignoredTypes);
   const schemaCache = new WeakMap<GraphQLSchema, GraphQLSchema>();
 
+  // never cache Introspections
+  ttlPerSchemaCoordinate = { ...ttlPerSchemaCoordinate, 'Query.__schema': 0 };
```

Comment:
```suggestion
  ttlPerSchemaCoordinate = { 'Query.__schema': 0, ...ttlPerSchemaCoordinate };
```

Just in case someone actually wants to introduce TTL for `Query.__schema` - Then he can override the default value of 0 or even enable long-living introspection caching by setting the value to `undefined` 🤔